BUILDING.md

```diff
@@ -42,7 +42,7 @@ platforms in production.
 | GNU/Linux    | Tier 1       | kernel >= 2.6.32, glibc >= 2.12  | x64, arm             |                  |
 | GNU/Linux    | Tier 1       | kernel >= 3.10, glibc >= 2.17    | arm64                |                  |
 | macOS        | Tier 1       | >= 10.10                         | x64                  |                  |
-| Windows      | Tier 1       | >= Windows 7/2008 R2/2012 R2     | x86, x64             | vs2017           |
+| Windows      | Tier 1       | >= Windows 7/2008 R2/2012 R2     | x64_x86, x64         | vs2017           |
```

Comment:
`x86_x86` was a typo
`x64_x86` is Microsoft designation for cross compiling x86 (32bit) target, on an x64(64bit) host.

```
calling: "C:\Program Files\VSCommunity2017\VC\\Auxiliary\Build\vcvarsall.bat" x86_x86
[ERROR:vcvarsall.bat] Invalid argument found : x86_x86
[ERROR:vcvarsall.bat] Error in script usage. The correct usage is:
Syntax:
    vcvarsall.bat [arch] [platform_type] [winsdk_version] [-vcvars_ver=vc_version]
where :
    [arch]: x86 | amd64 | x86_amd64 | x86_arm | x86_arm64 | amd64_x86 | amd64_arm | amd64_arm64
```

Reply:
"x64_x86" is a Microsoft-specific thing, it will confuse people. They expect the Architectures column to list the architecture on which Node runs, not how it's built.

Reply:
I also think this is confusing. IMHO we should keep this as it is, and add the note about `x64` host.